app/assets/javascripts/surveys.js

```diff
@@ -305,6 +305,39 @@
           ]
         },
         allowedOnMobile: true
+      },
+      {
+        identifier: 'CTTUK_iteration 8',
```

Comment:
This is the only thing that I am uncertain about that perhaps we should clarify whether the identifier should be `CTTUK_iteration_8`. The field seems to be only used for analytics, so perhaps this needs to link to something there?